openpype/hosts/maya/plugins/publish/validate_rig_output_ids.py

```diff
@@ -53,7 +57,20 @@ def get_invalid_matches(cls, instance, compute=False):
                 return invalid
 
             instance_nodes = cmds.sets(out_set, query=True, nodesOnly=True)
+
+            skeletonMesh_set = instance.data["rig_sets"].get(
+                "skeletonMesh_SET")
+            if not skeletonMesh_set:
+                instance.data["mismatched_output_ids"] = invalid
+                return invalid
+            else:
+                skeletonMesh_nodes = cmds.sets(
+                    skeletonMesh_set, query=True, nodesOnly=True)
+                instance_nodes += skeletonMesh_nodes
```

Comment:
This doesn't work - the `compute` toggle won't work for the different sets because there's only one `mismatched_output_ids` cache, etc. 

This should be done differently and you're likely best of initially already combining all members you need to query together.

If there's no skeleton mesh set this will now never validate and always return `{}`

Reply:
I will try to put all these changes into separate validators